test/traversals/dfs.jl

```diff
@@ -26,8 +27,9 @@
 
   @testset "is_cyclic" begin
     for g in testgraphs(path_graph(2))
-      @test @inferred(is_cyclic(g))
+      @test @inferred(!is_cyclic(g)) 
```

Comment:
Why the `!`?

Reply:
If g is a ```path_graph(2)``` the output of ```is_cyclic(g)``` is false. There are no cycles in a path.  
Hence the test could be: 
```Julia
@test @inferred (is_cyclic(g)==false)
```
or 
 ``` Julia
@test @inferred(!is_cyclic(g)) 
```
I used the second version because I saw it in the code.